src/db/http/docusign-dao.js

```diff
@@ -0,0 +1,50 @@
+import config from 'config';
+import createError from 'http-errors';
+import rp from 'request-promise-native';
+
+import { serializeTabs } from 'serializers/docusign-serializer';
+import { parseQuery } from 'utils/parse-query';
+import { paramsLink } from 'utils/uri-builder';
+import { httpOptions } from './connection';
+
+const { baseUri } = config.get('dataSources.http');
+
+/**
+ * Return a list of pets
+ *
+ * @param {object} params Request parameter object
+ * @param {object} query Request query object
+ * @returns {Promise} Promise object represents a list of pets
+ */
+const getEnvelopeDocumentTabs = async (params, query) => {
+  const { envelopeId, documentId } = params;
+  const { pageNumbers } = parseQuery(query);
+
+  let rawTabs;
+  try {
+    let uri = `${baseUri}/envelopes/${envelopeId}/documents/${documentId}/tabs`;
+
+    if (pageNumbers) {
+      uri = paramsLink(uri, { page_numbers: pageNumbers.join(',') });
+    }
+
+    rawTabs = await rp.get({ ...{ uri }, ...httpOptions });
+    return serializeTabs(rawTabs, params, query);
```

Comment:
I think we should call the serializer from the endpoint instead of the dao like we did for persons

Reply:
Agree. Fixed.